docs/docs/recipes.md

```diff
@@ -131,6 +131,54 @@ Showtime.
 
 ## Querying data
 
+### Using PageQuery
+
+You can use the `graphql`-tag to query data in your pages.
+
+#### Directions
+
+1. Import `graphql` from `gatsby`.
+
+2. Export a constant named `query` and set its value to be a `graphql` [tagged template](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) with the query between two backticks.
+
+3. Destruct `data` from props.
```

Comment:
```suggestion
3. Pass in `data` as a prop to the component.
```
I think the terminology could be simplified here so that people aren't thrown off by destructuring. Either that or we'll have to explain it.